rails/es6.md

```diff
@@ -0,0 +1,105 @@
+## [Home](../README.md)
+
+## Summary
+
+Including ES6 in Rails applications can be very useful. Using JavaScript
+packages that are only available/maintained in npm version has traditionally
```

Comment:
Would change `available/maintained in npm version` to `available/maintained in npm`